src/components/content.js

```diff
@@ -144,14 +144,35 @@ class Content extends React.Component {
   }
 
   /**
+   * The React ref method to set the root content element locally.
+   *
+   * @param {Element} n
+   */
+
+  ref = (n) => this.element = n
+
+  /**
+   * Check if an `event` is being fired from inside a non-contentediable child
+   * element, in which case we'll want to ignore it.
+   *
+   * @param {Event} event
+   * @return {Boolean}
+   */
+
+  isNonEditable = (event) => {
+    const { target } = event
+    return !target.isContentEditable || target !== this.element
```

Comment:
Does this need to check for whether the target element is **inside** `this.element`? Or is it only ever going to fire with the `contenteditable` element as the target. Asking because I can't remember, but just want to double check we're not leaving a gap here.